src/apps/statements/expressPresenter/utils/validateStatementBypassQueues.ts

```diff
@@ -0,0 +1,13 @@
+import { createWarning, Warnings } from 'rulr';
+
+export const validateStatementBypassQueues = (
+  statementBypassQueues: string[] | undefined,
+): void | Warnings[] => {
+  if (
+    statementBypassQueues &&
+    !statementBypassQueues.every((queueName) => queueName.startsWith('STATEMENT_'))
```

Comment:
Nit: We may want a TODO comment here to improve this if we move the xapi-service code into Enterprise where we could validate against the actual queue names.

Reply:
@ryansmith94 , me and @lucuilearningpool wanted to do that but we had concerns about the risk of exposing actual queue names in the open source repo